src/components/Card/CardSkeleton/CardSkeleton.js

```diff
@@ -3,39 +3,54 @@
  * @copyright IBM Security 2019
  */
 
+import classnames from 'classnames';
+import { PropTypes } from 'prop-types';
 import React from 'react';
 
 import { getComponentNamespace } from '../../../globals/namespace/index';
 
-import Card from '..';
+import Card from '../Card';
 import SkeletonText from '../../SkeletonText';
 
 const namespace = getComponentNamespace('card--skeleton');
 
-const WIDTHS = {
-  sm: '25%',
-  md: '50%',
-  lg: '75%',
-};
-
-const { sm, md, lg } = WIDTHS;
-
 /**
  * Card skeleton component.
  */
-const CardSkeleton = () => (
-  <Card className={namespace}>
-    <SkeletonText width={sm} />
-    <SkeletonText width={sm} heading />
-    <SkeletonText width={md} />
-    <SkeletonText width={md} />
-    <SkeletonText width={lg} />
-
-    <div className={`${namespace}__wrapper`}>
-      <SkeletonText className={`${namespace}__text--inline`} width={sm} />
+function CardSkeleton({ className, ...other }) {
+  const classes = classnames(namespace, className);
+
+  const widths = {
+    sm: '25%',
+    md: '50%',
+    lg: '75%',
+  };
+
+  const { sm, md, lg } = widths;
```

Comment:
Didn't make sense to keep these as "globals" in this file -- they aren't being used anywhere else.